src/common/minting/commands/common/mint-nft-command.spec.ts

```diff
@@ -79,7 +79,7 @@ describe('MintNftCommand', () => {
         const state = { cid: '123' } as MintState;
         const expected = {
           ...state,
-          error: MintError.RequiredRoyalty,
+          error: MintError.RequiredMintEventName,
```

Comment:
this is not correct. You should account for the new variable and keep the royalty one